Curtains/Curtains.Api/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Curtains.Application.Interfaces;
+using Curtains.Application.DTO;
+using Curtains.Domain.Models;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Curtains.Api.Controllers
+{
+    /// <summary>
+    /// This controller <c> ReviewController </c> for CRUID operations in database
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ReviewController : ControllerBase
+    {
+        private readonly IReviewService _service;
+
+        public ReviewController(IReviewService service)
+        {
+            _service = service;
+        }
+
+        /// <summary>
+        /// This method handles requests and get all Reviews from database.
+        /// </summary>
+        /// <returns> Http status code 200 </returns>
+        [HttpGet]
+        public ActionResult<IEnumerable<ReviewDTO>> GetAll()
+        {
+            var model = _service.GetAll();
+            return Ok(model);
+        }
+
+        /// <summary>
+        /// This method handles requests and get Review from database by entity Id.
+        /// </summary>
+        /// <param name="id"> Guid Reviews entity idetifier </param>
+        /// <returns> Http status code 200 </returns>
+        [HttpGet("~/Id")]
```

Comment:
I think it's better to use [HttpGet({id})] to save default route

Reply:
or this code do the same?

Reply:
like '/api/reviewController/2'

Reply:
Changed to ({id})